# @node-versions: v24.10.0+
# @description: Fix GCC LTO configuration for compatibility
#
# Removes -fuse-linker-plugin flag that causes issues with some GCC versions

--- a/common.gypi
+++ b/common.gypi
@@ -192,7 +192,7 @@
             ['clang==1', {
               'lto': ' -flto ', # Clang
             }, {
-              'lto': ' -flto=4 -fuse-linker-plugin -ffat-lto-objects ', # GCC
+              'lto': ' -flto=4 -ffat-lto-objects ', # GCC
             }],
           ],
         },
